### ISSUE TITLE
Digest not removed on time or never

### ISSUE THREAD

**Issue by blanchma** [mhenrixon/sidekiq-unique-jobs#659]
**Describe the bug**
Digest created 30 or more minutes are not removed.

**Expected behavior**
Only see short lived digest

**Current behavior**
Digests with "since" of hours

**Worker class**
SidekiqUniqueJobs.configure do |config|
  config.logger          = Sidekiq.logger
  config.logger.level    = Logger::DEBUG unless ENV["RACK_ENV"] == "production"
  config.debug_lua       = false # Turn on when debugging
  config.lock_info       = true # Turn on when debugging
  config.lock_ttl        = 5 * 60 # Expire locks after 5 minutes
  config.lock_timeout    = 5     # turn off lock timeout
  config.max_history     = 0     # Turn on when debugging
end

```ruby
class MyWorker
  include Sidekiq::Worker
  sidekiq_options lock: :until_executed, queue: :undefault
  def perform(args); end

  def self.lock_args(args)
    # the way you consider unique arguments
  end
end
```

**Additional context**

```
sidekiq (6.3.1)
      connection_pool (>= 2.2.2)
      rack (~> 2.0)
      redis (>= 4.2.0)
    sidekiq-unique-jobs (7.1.8)
      brpoplpush-redis_script (> 0.1.1, <= 2.0.0)
      concurrent-ruby (~> 1.0, >= 1.0.5)
      sidekiq (>= 5.0, < 8.0)
      thor (>= 0.20, < 3.0)
```

![image](https://user-images.githubusercontent.com/423378/142403725-4fbae157-a444-4a59-be9f-617db17777d1.png)

**Comment by mhenrixon** [comment]
Closing in favor of #653

**Comment by mhenrixon** [comment]
This has been fixed in `v7.1.1` @blanchma
